.github/workflows/publish-to-npm.yml

```diff
@@ -131,7 +131,7 @@ jobs:
           It must be MERGED into dev, NOT SQUASHED OR REBASED. Squashing or rebasing this branch onto dev can cause potentially irreconcilable merge conflicts later.\n\
           As an additional safeguard and reminder, the title of this PR MUST include the word 'merging' in the description portion of the PR title, e.g., 'Main2Dev @W-XXXXXXX@ Merging main to dev after vX.Y.Z'.\n\
```

Comment:
Should this be updated to say "dev-5" and main-5... etc. Same as line 125, line 138,  and possibly others?